SCClassLibrary/Common/GUI/Base/QWindow.sc

```diff
@@ -104,7 +104,16 @@ Window {
 		resizable = true,
 		border = true,
 		server,
-		scroll = false;
+		scroll = false,
+		forceCreate = false;
+
+		if( forceCreate.not, {
+			var existingWindows = Window.allWindows.select({|w| w.name == name});
```

Comment:
if you like, you could replace this by 

`alreadyExists = Window.allWindows.any({|w| w.name == name })`

Reply:
I would have sought for something like `SELECT * FROM windows WHERE name='' LIMIT 1`.
Using any here is indeed clearer but one needs to scan over the window array two times this way - with the approach above there is only one scan needed - but I am happy to change this for readability.